### PR TITLE
[RFC] "Yocto" migration to v4.14.75-ltsi/rcar-3.9.2

### PR DESCRIPTION
**Pull request by otyshchenko1**
This must be in sync with "Linux" migration to v4.14.75-ltsi/rcar-3.9.2.

Please see https://github.com/xen-troops/linux/pull/63

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-dom0.dts [hunk 1]
```diff
@@ -52,11 +52,10 @@
 	 * CPU nodes to be present get broken. This is why we have to explicitly
 	 * remove the following.
 	 */
-	soc {
-		/delete-node/thermal-zones;
-		/delete-node/pmu_a57;
-		/delete-node/pmu_a53;
-	};
+	/delete-node/thermal-zones;
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-dom0.dts]: For the whole patch: it does need description and rationale for what is done

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/displaymanager/displaymanager_git.bbappend [hunk 2]
```diff
@@ -1,4 +1,4 @@
-SRCREV = "6cbf0acec5d33a3be683e48a9cd4be4d16275fe5"
+SRCREV = "${AUTOREV}"
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/displaymanager/displaymanager_git.bbappend]: For the commit message: do not use "unfix", but simply state we are moving to AUTOREV here

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux-libc-headers/linux-libc-headers_%.bbappend [hunk 3]
```diff
@@ -1,5 +1,9 @@
 RENESAS_BSP_URL = "git://github.com/xen-troops/linux.git"
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux-libc-headers/linux-libc-headers_%.bbappend]: WHY is this patch if it is marked as [do not merge]????

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-h3ulcb.cfg [hunk 4]
```diff
@@ -33,7 +33,6 @@ dt_compatible = [ "renesas,h3ulcb", "renesas,r8a7795" ]
 # Clocks and regulators are needed and are in the device tree root,
```
Comment (andr2000) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-h3ulcb.cfg]: This patch does need explanation and rationale for the changes

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/displaymanager/displaymanager_git.bbappend (hunk 2 above)
Comment (otyshchenko1) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/displaymanager/displaymanager_git.bbappend]: ok

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux-libc-headers/linux-libc-headers_%.bbappend (hunk 3 above)
Comment (otyshchenko1) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux-libc-headers/linux-libc-headers_%.bbappend]: I will remove it from PR

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-h3ulcb.cfg (hunk 4 above)
Comment (otyshchenko1) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domd-h3ulcb.cfg]: ok, will explain in details

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-dom0.dts (hunk 1 above)
Comment (otyshchenko1) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-dom0.dts]: ok, will do

Comment (otyshchenko1) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-dom0.dts]: ok, will provide

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Pull request merged**